index.html

```diff
@@ -90,6 +90,8 @@
 
 			<!--Drop down controls-->
 
+			<input id="school-districts-text-entry" name="school-districts-text-entry" type="text" placeholder="enter ISD name" oninput="textZoomHandler('isd', 'state-school-districts', 'NAME', this.value);" />
```

Comment:
I'm sure most people using this map will know what ISD stands for, but it might be more intuitive to say "enter District name" here